CMakeLists.txt

```diff
@@ -885,7 +885,7 @@ cmake_dependent_option(
   "USE_ROCM"
   OFF)
 
-IF(USE_FBGEMM_GENAI AND USE_ROCM AND NOT "gfx942" IN_LIST PYTORCH_ROCM_ARCH)
+IF(USE_FBGEMM_GENAI AND USE_ROCM AND NOT ("gfx942" IN_LIST PYTORCH_ROCM_ARCH OR "gfx950" IN_LIST PYTORCH_ROCM_ARCH))
```

Comment:
We should only turn on fbgemm for 942, so 950 related things could be removed from here as well.